pkg/backends/statsdaemon/tls.go

```diff
@@ -0,0 +1,56 @@
+package statsdaemon
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"io/ioutil"
+)
+
+func getTLSConfiguration(caPath, certPath, keyPath string, enable bool) (*tls.Config, error) {
+	if !enable {
+		return nil, nil
+	}
+
+	tlsConfig := &tls.Config{
+		MinVersion: tls.VersionTLS12,
+		// Rationale:
+		// * Retain performance-conscious ordering from the crypto/tls defaults
+		// * Reject deprecated ciphers and modes, including non-DHE
```

Comment:
Hm, this is interesting. I'd like us to either omit cipher configuration completely or specify it for all other HTTPS clients too (AWS and Datadog IIRC).